pkg/model/image_manager.go

```diff
@@ -18,7 +18,7 @@ const (
 	DefaultKeycloakImage         = "quay.io/keycloak/keycloak:latest"
 	DefaultRHSSOImageOpenJ9      = "registry.redhat.io/rh-sso-7/sso74-openj9-openshift-rhel8:7.4"
 	DefaultRHSSOImageOpenJDK     = "registry.redhat.io/rh-sso-7/sso74-openshift-rhel8:7.4"
-	DefaultKeycloakInitContainer = "quay.io/keycloak/keycloak-init-container:master"
+	DefaultKeycloakInitContainer = "quay.io/keycloak/keycloak-init-container:12.0.4"
```

Comment:
I don't think we should hardcode the init container version, at least not in upstream/master. Shouldn't we rather fix the `master` tag?

Reply:
I feel pinning a stable version rather than using a floating tag is a better for the stability of the keycloak-operator. If you're using a floating tag, you can have the functionality of a container change from under you at any point. You can run full integration tests, verify everything works, publish a stable release, and only after some indeterminate time have all your stable releases break because a new incompatible version of a dependency image is published.

In the specific case of the image defined here, if a new master tag image had been published in the last year, it would have caused keycloak-operator to immediately stop being able to load extensions in all cases, due to a breaking change that was introduced in this PR:

https://github.com/keycloak/keycloak-containers/commit/da4b38c469077b78b46d8b18cf5f97ed8910f520#diff-51b1ab20d6e750a4464ec80bb5021a47be580d89ccc4cd25427ee812c5893347L33-R44

By extension, if we fixed the master tag as you are suggesting, it would also immediately break every keycloak-operators.

For these reasons, I strongly suggest using the pinned stable version rather than the floating tag, and ending the use of the master tag for images in the keycloak-containers repository.